core/store/models/service_agreement.go

```diff
@@ -209,7 +212,11 @@ func (e Encumbrance) ABI() ([]byte, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "while writing oracles")
 	}
-	_, err = buffer.Write(address256Bits(e.Aggregator))
+	_, err = buffer.Write(digest.Bytes())
```

Comment:
I think we should remove `requestDigest` from the message to be hashed. I don't think it achieves anything, but makes the protocol more fragile and difficult to implement. I suppose it might make finding a service-agreement preimage to an already-signed hash trickier, if keccak is ever broken, but I think that's unlikely.

Should be easy to remove: just take it out of here, `Coordinator#getId`, `calculateSAID2` in `helpersV2.ts`, and the tests which explicitly construct the signee message here and in `Coordinator_test.js`.

Reply:
It proves to each oracle that they have all agreed to the same off-chain work. I think we should keep it, and if we do decide to remove it that doesn't need to happen in this PR.